### PR TITLE
Adding P102 (Semimetrizable) to S60 (Pointed rational extension of reals) and S61 (Pointed irrational extension of reals)

### PR DESCRIPTION
**Pull request by david20000813**

### CODE REVIEW

File: spaces/S000060/properties/P000102.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+property: P000102
+value: true
+refs:
+- mathse: 5016051
+  name: Semimetric on pointed rational extension of reals
+- mathse: 5016298
+  name: Semimetrizability of pointed irrational/rational extensions of reals
+---
+
+See {{mathse: 5016051}} and {{mathse: 5016298}}.
```
Comment (yhx-12243) [inline spaces/S000060/properties/P000102.md:12]:
```suggestion
See {{mathse:5016051}} and {{mathse:5016298}}.
```

File: spaces/S000061/properties/P000102.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+space: S000061
+property: P000102
+value: false
+refs:
+- mathse: 5016298
+  name: Semimetrizability of pointed irrational/rational extensions of reals
+---
+
+See {{mathse: 5016298}}.
```
Comment (yhx-12243) [inline spaces/S000061/properties/P000102.md:10]:
```suggestion
See {{mathse:5016298}}.
```

**Comment by pzjp** [comment]
I see effects of my activity :)

**Comment by prabau** [comment]
For S60 (pointed rational extension of the reals), it would seem that https://math.stackexchange.com/questions/5016298 would be sufficient.

@pzjp, what do  you think?

**Comment by pzjp** [comment]
Technically, yes. The initial post is also referenced therein. On the other hand, the general proof is a bit more complicated so I imagine someone preffering my post for of a quick argument behind P102 for S60.

**Comment by prabau** [comment]
@pzjp Well, ok.  But then we should mention that there are various things in your post that need clarification.  Some of terminology needs to be checked to match pi-base.  Also, this "convergence characterization" vs. the easier definition by neighborhood bases at each point may need to be discussed.  I have not looked in detail.
What do you think?

**Comment by prabau** [comment]
@pzjp Looking at your mathse post a little further, it is phrased in an ambiguous way, so it is not clear what the result is, most probably as a way to solicit answers. But it does not seem that it's going to get any answer.

If you want to make it usable for pi-base, one suggestion would be edit the question to only ask if the space in question is semimetrizable.  And then add an answer explaining without any doubt why it is.

**Comment by pzjp** [comment]
Ok. My post is not that complete. [Q.5016298](https://math.stackexchange.com/questions/5016298)resolves both cases so it can be the lone reference. In fact filling the details in my reasoning would lead to a similar proof so there is no need to post it separately.

File: spaces/S000060/properties/P000102.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+space: S000060
+property: P000102
+value: true
+refs:
+- mathse: 5016051
```
Comment (david20000813) [inline spaces/S000060/properties/P000102.md:6]:
```suggestion
```

File: spaces/S000060/properties/P000102.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  name: Semimetric on pointed rational extension of reals
```
Comment (david20000813) [inline spaces/S000060/properties/P000102.md:7]:
```suggestion
```

File: spaces/S000060/properties/P000102.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+- mathse: 5016298
+  name: Semimetrizability of pointed irrational/rational extensions of reals
+---
+
+See {{mathse:5016051}} and {{mathse:5016298}}.
```
Comment (david20000813) [inline spaces/S000060/properties/P000102.md:12]:
```suggestion
See {{mathse:5016298}}.
```

**Comment by david20000813** [comment]
The references are now changed according to suggestions. (Btw, I’m aware of batching suggestions and committing simultaneously this time, but I don’t have my laptop near me at the moment and am instead working on my iPad, where, for some reason, I can batch suggestions but because of a display bug, I can’t commit the batched suggestions. So I have to commit one-by-one this time.)

**Comment by prabau** [comment]
@david20000813 One other technical detail.  If one wants to modify two lines as part of a review, one can select both lines in the left margin and then give the suggested change, instead of doing it one line at a time.

**Review by prabau — APPROVED** [review]

**Comment by david20000813** [comment]
> @david20000813 One other technical detail. If one wants to modify two lines as part of a review, one can select both lines in the left margin and then give the suggested change, instead of doing it one line at a time.

I’ll try that the next time I’m working on my laptop. GitHub really does not work well with my iPad.

**Pull request merged**